proto/proto-backplane-grpc/src/main/proto/deephaven/proto/table.proto

```diff
@@ -315,8 +315,14 @@ message EmptyTableRequest {
 
 message TimeTableRequest {
   Ticket result_id = 1;
-  sint64 start_time_nanos = 2 [jstype=JS_STRING];
-  sint64 period_nanos = 3 [jstype=JS_STRING];
+  oneof start_time {
+    uint64 start_time_nanos = 2 [jstype = JS_STRING];
+    string start_time_string = 4;
+  }
+  oneof period {
+    sint64 period_nanos = 3 [jstype = JS_STRING];
+    string period_string = 5;
+  }
```

Comment:
This look structurally good. I'm wondering if we want to update these names so they don't have redundant prefixes?

Also, I wonder if we shouldn't change the name "period" to "duration" since this is _really_ a duration and not a period (wrt the time library terminology).

Reply:
The names of the `oneof` structures aren't propagated to the setter/getter functions of the Java/Python structures, so I think we need the specific names.

On the second point, I'm ambivalent.  `period` and `duration` seem equally accurate in this context, or even `interval` given the the nature of the time table being created.